tests/unit/FindCacheTest.php

```
@@ -0,0 +1,438 @@
+
+    public function ping(): bool
+    {
+        return true;
+    }
+
+    public function getSize(): int
+    {
+        return \count($this->store);
+    }
+
+    public function getName(?string $key = null): string
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$key'. (undefined)

(UnusedFormalParameter)